### ISSUE TITLE
Available to have multiple draggable container and multiple droppable container? 

### ISSUE THREAD

**Issue by cuongdevjs** [clauderic/dnd-kit#351]
I think you should implement some examples on documentation about this.

**Comment by quacrobat** [comment]
+1

**Comment by slugmandrew** [comment]
Hey, fellow learner here. There are some pretty good examples on the storybook, the source of which is in this repo.

In particular the Checkers one is quite good for showing multiple draggables / droppables together.

**Comment by sairion** [comment]
There is pretty great demos in storybook, but it is pretty hard to follow IMO. I think minimal example would be a good start.

**Comment by KillerKiwiJuice** [comment]
> There is pretty great demos in storybook, but it is pretty hard to follow IMO. I think minimal example would be a good start.

I agree. This package is amazing, but it's incredibly difficult to learn anything from these storybook examples.

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/guides/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
